Sources/OpenAPIURLSession/URLSessionTransport.swift

```diff
@@ -90,7 +90,18 @@ public struct URLSessionTransport: ClientTransport {
     }
 
     private func invokeSession(_ urlRequest: URLRequest) async throws -> (Data, URLResponse) {
-        #if canImport(FoundationNetworking)
+    #if canImport(FoundationNetworking)
+        return try await performDataTask(with: urlRequest)
+    #else
+        if #available(iOS 15.0, *) {
```

Comment:
Should this also include the macOS, tvOS, and watchOS versions aligned with iOS 15?